### PR TITLE
Update stable version to TiSpark 2.1

### PR DESCRIPTION
**Pull request by birdstorm**
After this update, we should: update all documents and scripts that rely on latest TiSpark stable version or master version. Including docs&docs-cn/ansible/docker deployment/etc.,.....

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,50 +11,43 @@ TiSpark is a thin layer built for running Apache Spark on top of TiDB/TiKV to an
...
-    </dependency>
-    <dependency>
-      <groupId>com.pingcap.tikv</groupId>
-      <artifactId>tikv-client</artifactId>
-      <version>2.0</version>
+      <version>2.1-spark_${spark.version}</version>
     </dependency>
 </dependencies>
 ```
-If you're using SBT, add the following line to your build file:
```
Comment (birdstorm) [inline README.md]: SBT no longer documented (per @zhexuany)

File: README.md [hunk 2]
```diff
@@ -110,9 +103,6 @@ spark.sql("select count(*) from lineitem").show
 spark.sql("select ti_version()").show
 ```
 
-## TiDB Data Source API
-When using the TiDB Data Source API, please follow the document for [TiDB Data Source API User Guide](./docs/datasource_api_userguide.md).
```
Comment (marsishandsome) [inline README.md]: please do not delete

File: core/pom.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>com.pingcap.tispark</groupId>
         <artifactId>tispark-parent</artifactId>
-        <version>2.1-SNAPSHOT</version>
+        <version>2.2-SNAPSHOT</version>
```
Comment (marsishandsome) [inline core/pom.xml]: may be `2.2.0-SNAPSHOT` is better

Comment (birdstorm) [reply core/pom.xml]: right.

File: README.md (hunk 2 above)
Comment (birdstorm) [reply README.md]: that was a mistake

**Pull request merged**